index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;800&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <section class="container">
+      <header>
+        <img src="images/Light.png" alt="status bar" width="100%" />
+        <div class="toolbar">
+          <h1 id="logo">ROZH</h1>
+          <img id="menu-image" src="images/Icon - Menu.png" alt="Menu icon" />
+        </div>
+      </header>
+
+      <section class="headline">
+        <article class="self-info">
+          <h2>
+            I'm Rozh <br />
+            Glad to see <br />
+            you! <br />
+          </h2>
```

Comment:
- [ ] This is the main header of the portfolio and should be wrapped in an `<h1>` tag. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info about the importance of the `h1` tag. Kindly change that.

Reply:
Thanks so much for the feedback. I changed it to h1.